src/Player.css

```diff
@@ -0,0 +1,21 @@
+
+.player{
+    display: flex;
+    background-color: white;
+    border: 1px solid lightgray;
+    margin-bottom: 10px;
+    min-width: 250px;
+}
+.playerText{
+    margin-top: 10px;
+    margin-bottom: 10px;
+    margin-left: 20px;
+    font-weight: normal;
+    padding: 10px;
+}
+
+.playerAvatar{
```

Comment:
Same thought here on the spacing. We should aim for consistency

Reply:
Resolved